src/SFA.DAS.Commitments.Support.SubSite/Views/Apprenticeships/CohortSearchSummary.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@using SFA.DAS.Commitments.Support.SubSite.Models
+@model CommitmentSummaryViewModel
+@{
+    ViewBag.Title = "CohortSearchSummary";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <h1 class="heading-xlarge" id="commitmentsHeader">View cohort</h1>
+    </div>
+</div>
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <table class="responsive">
+            <thead>
+                <tr>
+                    <th colspan="2">
+
+                        <p>
+                            We found the following cohort:
+                        </p>
+
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Cohort reference</td>
+                    <td>@Model.CohortReference</td>
+                </tr>
+                <tr>
+                    <td>Employers account name</td>
```

Comment:
Apostrophe needed? Also, technically, this is the legal entity name

Reply:
Good spot.